packages/blade/src/components/Amount/_decisions/decisions.md

```diff
@@ -0,0 +1,75 @@
+# Amount <!-- omit in toc -->
+
+Amount component is a UI element that displays and formats various currency values in a specified format.
+
+<img  src="./amount-thumbnail.png"  width="380" />
+
+## Design
+
+- [Figma - Amount](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=28012%3A580578&t=3peAz8A2n2Gw4WMl-1)
+
+## API
```

Comment:
Let's also get the props synced to figma. Could you please followup with @saurav12 and ensure these are updated on the component figma?